jobs/src/main/scala/com/pennsieve/jobs/types/DeleteJob.scala

```diff
@@ -475,6 +478,7 @@ class DeleteJob(
           case ex: MultiObjectDeleteException =>
             val deletedIds = ex.getDeletedObjects.asScala
               .map(_.getKey)
+            deleteVersionsForKeys(s3Keys)
```

Comment:
what is the difference between the s3Keys and deletedKeys (deletedIds.toSeq) in this particular case?

On failure should we be doing the same thing we are doing in the `try` segment of the code?

Reply:
The S3keys are a string path to where the file lives, the deletedIds.toSeq is an ID result from the delete.

RE on failure block: yeah let me remove that